server/server_test.go

```diff
@@ -232,7 +232,7 @@ func TestParseAtlantisURL(t *testing.T) {
 			ExpErr: "http or https must be specified",
 		},
 		{
-			In:     "httpl://localhost/test",
+			In:     "http0://localhost/test",
```

Comment:
```suggestion
			In:     "httpl://localhost/test",
```

Reply:
this change doesn't make sense

Reply:
For `httpl:`, afaict you're just trying to use any typo, and `http0:` is just as much of a non protocol as `httpl:` it just isn't a misspelling.

Reply:
They're both made up protocols. I am averse to making code changes to satisfy a ci/cd tool that isn't in the project's pipeline. This just makes for extra work for the maintainer without any benefit to the project.

In this case since the large majority of the suggestions are good, I'm okay to keep this change to make your job easier.